### PR TITLE
Announcing data ingestion ports

### PR DESCRIPTION
**Pull request by achalpandey**

### CODE REVIEW

File: tigon-sql/src/main/java/co/cask/tigon/sql/flowlet/AbstractInputFlowlet.java [hunk 1]
```diff
@@ -317,8 +317,9 @@ public void notifyFailure(Set<String> errorProcessNames) {
   @Override
   public void announceReady() {
     FlowletContext ctx = getContext();
-    for (Map.Entry<String, Integer> portEntry : dataIngestionPortsMap.entrySet()) {
-      portsAnnouncementList.add(ctx.announce(portEntry.getKey(), portEntry.getValue()));
+    for (String key : dataIngestionPortsMap.keySet()) {
+        portsAnnouncementList.add(ctx.announce(key, inputFlowletService.getDataPort(key)));
+      LOG.info("Announced Data Port {} - {}", key, inputFlowletService.getDataPort(key));
```
Comment (gokulavasan) [inline tigon-sql/src/main/java/co/cask/tigon/sql/flowlet/AbstractInputFlowlet.java:322]: Alignment

**Comment by gokulavasan** [comment]
Small comment. Otherwise, LGTM :+1:

File: tigon-sql/src/main/java/co/cask/tigon/sql/ioserver/StreamEngineIO.java [hunk 2]
```diff
@@ -128,5 +130,9 @@ public void shutDown() {
       service.stopAndWait();
     }
   }
+
+  public int getDataPort(String key) {
```
Comment (chtyim) [inline tigon-sql/src/main/java/co/cask/tigon/sql/ioserver/StreamEngineIO.java:134]: Is this method only get called after the service is started?

Comment (achalpandey) [inline tigon-sql/src/main/java/co/cask/tigon/sql/ioserver/StreamEngineIO.java:134]: Yes, thats correct. This is called from an announceReady() function which gets invoked after all the services and the GSTool processes have been initiated

**Comment by gokulavasan** [comment]
@achalpandey can we merge this?

**Pull request merged**